src/main/java/com/dataloom/edm/EdmApi.java

```diff
@@ -183,6 +184,11 @@ Void removePropertyTypeFromEntityType(
     Schema getSchemaContents(
             @Path( NAMESPACE ) String namespace,
             @Path( NAME ) String name );
+    
+    @GET( SCHEMA_BASE_PATH + NAMESPACE_PATH + NAME_PATH + YAML_PATH )
+    String getSchemaContentsAsYaml(
```

Comment:
You can look at `loadEntitySetData` in `DataApi` and `DataController`. Be careful in the controller - to set content disposition you will need to include the `HttpServletResponse` that wouldn't override the API method. For that reason, the controller has one method that overrides the API method but with no handler attached, and another method with the handler but not overriding any methods.

The filetype enum is in DataApi.